daemon/oci_windows.go

```diff
@@ -326,12 +336,20 @@ func (daemon *Daemon) createSpecWindowsFields(c *container.Container, s *specs.S
 // TODO: @jhowardmsft LCOW Support. We need to do a lot more pulling in what can
 // be pulled in from oci_linux.go.
 func (daemon *Daemon) createSpecLinuxFields(c *container.Container, s *specs.Spec) error {
+	s.Root = &specs.Root{
```

Comment:
(just curious); would, at some point, LCOW be able to just take the "linux" spec, and do the translation of LCOW-specific options in `runhcs` ("ignore" / "warn" about options that aren't supported, and/or translate, e.g. `memory` -> hyper-v memory etc.?)

Reply:
In terms of a spec, generally yes. Using containerd for LCOW outside of moby does just that - it's a Linux spec run on a Linux utility VM so pretty much everything works (or will, at least - still some bugs being worked on in opengcs and containerd).

However, in terms of using the existing Linux code in moby to create a Linux spec, generally no. The current Linux code is too highly dependent on assuming the host is Linux, and much of the processing it does just wouldn't work on Windows.